conbench/entities/benchmark_result.py

```diff
@@ -300,7 +300,7 @@ def update(self, data):
 s.Index("benchmark_result_timestamp_index", BenchmarkResult.timestamp)
 
 
-class BenchmarkResultCreate(marshmallow.Schema):
+class BenchmarkResultStatsSchema(marshmallow.Schema):
```

Comment:
This diff line is the actual and rather significant motivator for this patch. The current state is a little too mind-boggling; and this here is a necessary change towards implementing a clear distinction between "benchmark" and "benchmark result".